mobile/lib/features/wallet/send/send_screen.dart

```diff
@@ -196,8 +197,8 @@ class _SendScreenState extends State<SendScreen> {
 
                   final amount = Amount.parseAmount(value);
 
-                  if (amount.sats <= 0) {
-                    return "Amount is mandatory";
+                  if (amount.sats < 0) {
```

Comment:
🔧 this will also impact paying a lightning invoice.

We need to check if the we are paying on-chain to relax that validation rule.

Reply:
fixed